### PR TITLE
Ibp 6119 implement new manage studies landing page deprecate browse studies

### PR DESCRIPTION
**Pull request by clarysabel**

### CODE REVIEW

File: src/main/java/org/generationcp/middleware/api/study/StudyDTO.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import org.pojomatic.Pojomatic;
 import org.pojomatic.annotations.AutoProperty;
 
+@Deprecated
```
Comment (clarysabel) [inline src/main/java/org/generationcp/middleware/api/study/StudyDTO.java:6]: @lik-83 if I am not wrong, we have said we will replace this class by the new one wherever it is used in a new ticket, right?

File: src/main/java/org/generationcp/middleware/service/impl/study/StudyServiceImpl.java [hunk 2]
```diff
@@ -189,12 +238,320 @@ public void deleteNameTypeFromStudies(final Integer nameTypeId) {
...
+		studyDetailsDTO.setName(studyData.getStudyName());
+		studyDetailsDTO.setDescription(studyData.getDescription());
+		studyDetailsDTO.setStudyType(studyData.getStudyTypeName());
+		studyDetailsDTO.setObjective(studyData.getObjective());
+		studyDetailsDTO.setCreatedByName(studyData.getOwnerName());
+		studyDetailsDTO.setStartDate(studyData.getStartDate());
+		studyDetailsDTO.setEndDate(studyData.getEndDate());
+		studyDetailsDTO.setLastUpdateDate(studyData.getUpdateDate());
+
+		// TODO: I added a method to retrieve a dataset with its variables in New Advance process implementation: https://github.com/IntegratedBreedingPlatform/Middleware/blob/0bbe6823d1c72f1cd4b3b14cd62f4e3c4e55fa8f/src/main/java/org/generationcp/middleware/service/api/dataset/DatasetService.java#L149
```
Comment (clarysabel) [inline src/main/java/org/generationcp/middleware/service/impl/study/StudyServiceImpl.java:281]: @lik-83 are these comments TODO's in fact or only comments?

File: src/main/resources/liquibase/workbench_changelog/23_1_0.xml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                 SELECT COUNT(*) FROM PERMISSION P WHERE P.NAME = 'BROWSE_STUDIES';
+             </sqlCheck>
+ 		</preConditions>
+		<sql>
+			update permission p1
+			    inner join permission p2  on p1.permission_id = p2.permission_id  and P2.name = 'BROWSE_STUDIES'
+			        set P1.name = 'VIEW_STUDIES', P1.description = 'View Studies', P1.workbench_sidebar_category_link_id=(SELECT sidebar_category_link_id FROM
+						workbench_sidebar_category_link where tool_name='study_manager_fieldbook_web');
+		</sql>
+         <comment>Migrate Browse Studies as View Studies permission</comment>
```
[the remark this answers is not in the document]
Comment (cuenyad) [reply src/main/resources/liquibase/workbench_changelog/23_1_0.xml]: done!

File: src/main/java/org/generationcp/middleware/dao/dms/StudySearchDAOQuery.java [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+				.findFirst()
+				.orElseThrow(() -> new IllegalStateException(String.format("Unsupported sort value %s.", value)));
+
+		}
+
+	}
+
+
+	//TODO: move to utils
+	public static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat(Util.DATE_AS_NUMBER_FORMAT);
```
[the remark this answers is not in the document]
Comment (lik-83) [reply src/main/java/org/generationcp/middleware/dao/dms/StudySearchDAOQuery.java:55]: Done!

**Pull request merged**